wireapp/wire-android
# feat: welcome to new android (AR-2118)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2118" title="AR-2118" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-2118</a>  Welcome to New Android - information to users when they just installed it
  </td></table>
  <br />
 












----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

After run a migration, we want to display for old users a brief information (only once) about what's new in the new version.

### Causes (Optional)

Displays a dialog only once after run the migration process.

### Solutions

Creates a flag, that will be read on first open, to identify if we need to display the info dialog or not to a migrated users.
For example, new users will not have this info.


### Testing

#### Test Coverage (Optional)


#### How to Test

Install the Scala app, then install on top the new version and see the dialog displayed.

### Attachments (Optional)

https://user-images.githubusercontent.com/5806454/203037969-eb44e254-abb3-4c97-95db-be21da38d42d.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
